packages/react-charting/src/utilities/utilities.ts

```diff
@@ -1350,10 +1450,11 @@ export const convertToLocaleString = (data: LocaleStringDataProps, culture?: str
   culture = culture || undefined;
   if (typeof data === 'number') {
     return data.toLocaleString(culture);
-  }
-  if (typeof data === 'string' && !window.isNaN(Number(data))) {
+  } else if (typeof data === 'string' && !window.isNaN(Number(data))) {
     const num = Number(data);
     return num.toLocaleString(culture);
+  } else if (data instanceof Date) {
+    return data.toLocaleDateString(culture);
```

Comment:
why is this change needed. It might break existing charts.

Reply:
The hover label is coming very long.

Reply:
Hover length is now proper.